supervised/loglizer/loglizer/dataloader.py

```
@@ -0,0 +1,268 @@
+            x_train, window_y_train, y_train = slice_hdfs(x_train, y_train, window_size)
+            x_test, window_y_test, y_test = slice_hdfs(x_test, y_test, window_size)
+            log = "{} {} windows ({}/{} anomaly), {}/{} normal"
+            print(log.format("Train:", x_train.shape[0], y_train.sum(), y_train.shape[0], (1-y_train).sum(), y_train.shape[0]))
+            print(log.format("Test:", x_test.shape[0], y_test.sum(), y_test.shape[0], (1-y_test).sum(), y_test.shape[0]))
+            return (x_train, window_y_train, y_train), (x_test, window_y_test, y_test)
+
+        if label_file is None:
+            if split_type == 'uniform':
+                split_type = 'sequential'
+                print('Warning: Only split_type=sequential is supported \
+                if label_file=None.'.format(split_type))
```

Ruff (0.8.2)

`.format` call has unused arguments at position(s): 0

Remove extra positional arguments at position(s): 0

(F523)